lang/cs/Org.Apache.REEF.Bridge/Clr2JavaImpl.h

```diff
@@ -158,12 +158,11 @@ namespace Org {
                             virtual void OnError(String^ message);
                             virtual IEvaluatorRequestorClr2Java^ GetEvaluatorRequestor();
                             virtual String^ GetId();
-                            virtual EvaluatorException^ GetException();
                             virtual array<IFailedContextClr2Java^>^ GetFailedContextsClr2Java();
                             virtual IFailedTaskClr2Java^ GetFailedTaskClr2Java();
-                        private:
-                            String^ GetCause();
-                            String^ GetStackTrace();
+                            virtual array<byte>^ GetErrorBytes();
+                            virtual String^ GetJavaCause();
+                            virtual String^ GetJavaStackTrace();
```

Comment:
Not necessarily. This was mainly used to indicate that the Cause and StackTrace are generated directly on the Java side.